examples/gsutil/outputs.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project_id" {
```

Comment:
Can we add descriptions to `project_id` and `region`?

Reply:
```
commit f83b4e6904218757c46a0f02cdb3421f02c5a551
Author: Jeff McCune <jeff@openinfrastructure.co>
Date:   Fri Feb 15 13:26:30 2019 -0800

    Fixup output descriptions

diff --git a/examples/gsutil/outputs.tf b/examples/gsutil/outputs.tf
index 0438282..de7cd18 100644
--- a/examples/gsutil/outputs.tf
+++ b/examples/gsutil/outputs.tf
@@ -15,11 +15,13 @@
  */
 
 output "project_id" {
-  value = "${var.project_id}"
+  description = "The project id used when managing resources."
+  value       = "${var.project_id}"
 }
 
 output "region" {
-  value = "${var.region}"
+  description = "The region used when managing resources."
+  value       = "${var.region}"
 }
 
 output "nat_ip" {

```

Reply:
@jeffmccune I'm not seeing that commit on this PR, could you make sure that you've pushed?